modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/exportimport/data/handler/FileEntryStagedModelDataHandler.java

```diff
@@ -939,6 +940,13 @@ private DDMFormValues _getImportDDMFormValues(
 		return DDMBeanTranslatorUtil.translate(ddmFormValues);
 	}
 
+	/**
+	 * @see com.liferay.portlet.documentlibrary.service.impl.DLFileEntryLocalServiceImplrviceImpl#_getStoreUUID(String)
+	 */
+	private String _getStoreUUID(String version) {
+		return version + StringPool.TILDE + UUID.randomUUID();
+	}
+
```

Comment:
The class name linked with @see mention is not correct.

Also, I don't like having duplicated code: can we avoid this case?

Reply:
> Change tracking requires the field to be stored in the database. That's why I removed the storeFileName computed value. There's no way out of that.

If the problem is the join with the CTSContent table, would it be something like this a viable solution?

* You could join tables by a concatenation, similarly to what is done in class `DLFileEntryTypeTableReferenceDefinition`: 
```java
			.innerJoinON(
				DLFileEntryTypeTable.INSTANCE,
				DLFileEntryTypeTable.INSTANCE.groupId.eq(
					DDMStructureTable.INSTANCE.groupId
				).and(
					DSLFunctionFactoryUtil.lower(
						DDMStructureTable.INSTANCE.structureKey
					).eq(
						DSLFunctionFactoryUtil.lower(
							DSLFunctionFactoryUtil.concat(
								new Scalar<>(
									DLUtil.getDDMStructureKey(
										StringPool.BLANK)),
								DLFileEntryTypeTable.INSTANCE.uuid))
					)
				)
			)
```

Reply:
I don't think so. This example embeds a literal value in the query; in our case, the join needs to be done with a value that needs to be computed for each row (the version number may be different for each row).